x/incentives/keeper/gauge.go

```diff
@@ -226,19 +228,6 @@ func (k Keeper) CreateGroupGauge(ctx sdk.Context, coins sdk.Coins, numEpochPaidO
 		return 0, fmt.Errorf("Invalid splitting policy, needs to be Evenly got %s", splittingPolicy)
 	}
 
-	// check that all the internalGaugeIds exist
-	internalGauges, err := k.GetGaugeFromIDs(ctx, internalGaugeIds)
-	if err != nil {
-		return 0, fmt.Errorf("Invalid internalGaugeIds, please make sure all the internalGauge have been created.")
-	}
-
-	// check that all internalGauges are perp
-	for _, gauge := range internalGauges {
-		if !gauge.IsPerpetual {
-			return 0, fmt.Errorf("Internal Gauge id %d is non-perp, all internalGauge must be perpetual Gauge.", gauge.Id)
-		}
-	}
-
```

Comment:
This was removed due to internal gauge ids no longer being a field that can be directly accessed. See issue in comment for more context on the intention with group gauge initialization logic (which will be done in a future PR)